k9mail/src/main/java/com/fsck/k9/helper/FileHelper.java

```diff
@@ -147,6 +141,11 @@ public static boolean move(final File from, final File to) {
 
         try {
             copyFile(from, to);
+
+            boolean deleteFromOk = from.delete();
```

Comment:
But `exists` is not called for its side effect that's why I kept the non-trivial expression outside the if there! aaahh :D

ok went with the trivial ifs